### DISCUSSION TITLE
Remember selected output tab between runs

### DISCUSSION THREAD

**Discussion by miv391** [godotengine/godot-proposals#4860] [Editor]
Currently when the program is started, the Output tab is always selected and opened. It would be better to remember which output tab (Output, Debugger, Audio, Animation or any subtab of those) was selected and not always change to the Output tab. For example, if I am monitoring whether nodes are properly freed, on every run I must first click the Debugger tab and then the Monitors tab. Or if I am fixing errors, it would be much better to see Errors tab all the time.

![kuva](https://user-images.githubusercontent.com/49998025/178452516-3262c597-bd41-420d-ad21-5dbcab67561d.png)

**Proposal**: Remember which tab was selected and do not always automatically select the Output tab at start. There could be an editor setting for this. My guess is that most users would like to keep their tab selection, so I would suggest that as a default action.

**Comment by xix-xeaon** [comment]
At least if the Debugger tab is open it should probably not switch to Output.

**Comment by azur-wolve** [comment]
+1
Also it remembers me how much a hassle is when you have more than one script in one scene tab, and you switch to another scene tab,
if you switch back to the previous, the script editor will switch to the script of the root scene (if it has one). 
So if you're working on another's scene node script, you must switch them manually over and over.

Sometimes I ended up transforming nodes into their own scene to just avoid that.
